eng/platform-matrix.yml

```diff
@@ -122,10 +188,20 @@ jobs:
     archType: x86
     osGroup: Windows_NT
     osIdentifier: Windows_NT
-    # TODO: add Windows.10.Nano.Amd64.Open when https://github.com/dotnet/coreclr/issues/21693 is resolved
-    # TODO: add Windows.7.Amd64.Open when https://github.com/dotnet/coreclr/issues/21796 is resolved
-    helixQueuesPublic: 'Windows.10.Amd64.Open,Windows.81.Amd64.Open'
-    helixQueuesInternal: 'Windows.10.Amd64,Windows.10.Nano.Amd64,Windows.10.Amd64.Core,Windows.7.Amd64,Windows.81.Amd64'
+    helixQueues:
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        # TODO: add Windows.7.Amd64.Open when https://github.com/dotnet/coreclr/issues/21796 is resolved
+        asString: 'Windows.10.Amd64.Open,Windows.81.Amd64.Open'
+        asArray:
+        - Windows.10.Amd64.Open
+        - Windows.81.Amd64.Open
+      ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+        asString: 'Windows.10.Amd64,Windows.10.Nano.Amd64,Windows.10.Amd64.Core,Windows.7.Amd64,Windows.81.Amd64'
+        asArray:
+        - Windows.10.Amd64
+        - Windows.10.Amd64.Core
```

Comment:
Missing Windows.10.Nano.Amd64, I think.

Reply:
I remove this on purpose for x86 - was originally added there by mistake